draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -326,6 +326,14 @@ http features such as {{?HPACK=RFC7541}}.
 
 A client MUST process the received `RateLimit` fields.
 
+A client SHOULD use a Structured Fields parser, but MAY reuse an existing parser,
+for example to migrate from legacy fields supporting only integer values.
+In doing so, they SHOULD:
```

Comment:
Does this apply to the first sentence "use a Structured Fields parser", or the second one "reuse an existing parser", or both? My assumption is it applies to the second so maybe we can reword it to make it explicit/more obvious?

Reply:
@unleashed feel free to reword :)